lib/rubocop/cop/layout/rescue_ensure_alignment.rb

```diff
@@ -30,7 +30,17 @@ class RescueEnsureAlignment < Cop
         ANCESTOR_TYPES = %i[kwbegin def defs class module].freeze
         RUBY_2_5_ANCESTOR_TYPES = (ANCESTOR_TYPES + %i[block]).freeze
         ANCESTOR_TYPES_WITH_ACCESS_MODIFIERS = %i[def defs].freeze
-        ASSIGNMENT_TYPES = %i[lvasgn].freeze
+        ASSIGNMENT_TYPES = %i[
```

Comment:
Instead of this, you can use `Node::ASSIGNMENTS`, defined in `node.rb`. It's quite helpful to keep these kinds of lists to a single place. 🙂

Reply:
Thank you @Drenmi, will do!

Reply:
Done! Using `node.assignment?` now as it has the same list of types.